src/pages/my_page/user_profile/ResearchProfilePanel.tsx

```diff
@@ -26,6 +28,18 @@ export const ResearchProfilePanel = ({ person, isLoadingPerson }: ResearchProfil
   const activeAffiliations = person?.affiliations ? filterActiveAffiliations(person.affiliations) : [];
   const personBackground = getLanguageString(person?.background);
   const personKeywords = person?.keywords ?? [];
+  const personId = person?.id ?? '';
+
+  const profilePictureQuery = useQuery({
+    queryKey: ['picture', personId],
+    queryFn: () => fetchProfilePicture(personId),
+    meta: { errorMessage: false },
+    retry: false,
+  });
+
+  const profilePictureString = profilePictureQuery.isSuccess
+    ? `data:image/jpeg;base64,${profilePictureQuery.data.base64Data}`
+    : '';
```

Comment:
Dette er (en for meg) mystisk ting som er implementert 3 ulike plasser nå. Antar det blir ønske om å vise bilde flere plasser også etter hvert, så kanskje lurt å lage en egen hook som gjøre dette, og gjenbruke den for alle plasser? Feks `useProfilePicture(personId)`

Reply:
Ser lenger ned at `<img>` også lages på nytt hver gang, men antar mye av det skal være felles, så kanskje enda bedre med en egen komponent, type `<ProfilePicture personId={...} />`.
Dette for å kunne løse eventuelle bugs bare en plass, i stedet for å trave gjennom alle plasser hvor vi etter hvert skal vise bilde

Reply:
Har en mistanke om at vi en dag går over til en enklere løsning hvor frontend bare peker til et bilde også, og om bruk av profilbilde går gjennom en felles komponent blir den endringsjobben mye enklere å ta

Reply:
Det kommer nok absolutt flere bilder (venstremeny, kurator-badges, header etc.). så er absolutt en god ide å mekke egen hook tenker jeg.

Reply:
Mtp skeleton som fallback er det kanskje enda bedre å tenke på det som en egen komponent, tror jeg